README.md

```diff
@@ -11,8 +11,14 @@ These crates are available from in (sdk):
 - azure_core
 - azure_identity
 - azure_cosmos
-- azure_storage
+- azure_data_tables
+- azure_iot_hub
+- azure_messaging_eventgrid
```

Comment:
I'd prefer to keep eventgrid out the readme until #254.

Reply:
Removed